runtime/src/main/java/org/corfudb/runtime/collections/CMap.java

```diff
@@ -0,0 +1,22 @@
+package org.corfudb.runtime.collections;
+
+import org.corfudb.annotations.Accessor;
+
+import javax.annotation.Nonnull;
+import java.util.Map;
+
+/**
+ * Created by box on 1/25/18.
+ */
+public interface CMap<K, V> extends Map<K, V>{
```

Comment:
You do not need this interface you can change ICorfuMap or CorfuTable

@Accessor
    public default AbstractMap.SimpleImmutableEntry<K, V>[] getEntries() {
        AbstractMap.SimpleImmutableEntry<K, V>[] entries = new AbstractMap.SimpleImmutableEntry[size()];
               int i = 0;
               for (Map.Entry<K, V> entry : entrySet()) {
                        entries[i++] = new AbstractMap.SimpleImmutableEntry<>(entry.getKey(), entry.getValue());
                    }
        return entries;
    }